### PR TITLE
🔥 Removing `construct_client_dict` by Fixing APIObject client attribute

### PR DESCRIPTION
**Pull request by Sigmanificient**

### CODE REVIEW

**Review by Endercheif — CHANGES_REQUESTED** [review]
Please run black on the files changed. I can tell you did not

File: pincer/objects/events/guild.py [hunk 1]
```diff
@@ -133,9 +132,9 @@ class GuildMemberRemoveEvent(APIObject, GuildProperty):
     user: User
 
     def __post_init__(self):
-        self.user = User.from_dict(
-            construct_client_dict(self._client, self.user)
-        )
+        # NOTE: Idk if this works, but I got a type Error otherwise
+        # TODO: Test this
+        self.user = User.from_dict({**self.user})
```
Comment (Lunarmagpie) [inline pincer/objects/events/guild.py:137]: concern

**Comment by Sigmanificient** [comment]
> Please run black on the files changed. I can tell you did not
Are you sure you want me to run black on 72 files?

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

File: pincer/objects/events/guild.py (hunk 1 above)
Comment (Sigmanificient) [reply pincer/objects/events/guild.py:137]: idk how to test it

**Review by trag1c — APPROVED** [review]

File: pincer/objects/events/guild.py (hunk 1 above)
Comment (Secrus) [reply pincer/objects/events/guild.py:137]: dirty trick, but `User.from_dict(vars(self.user))` should work

**Pull request merged**